esp/platform/cumulativetimer.hpp

```diff
@@ -41,11 +41,6 @@ class CumulativeTimer : public CInterface
     {
     }
 
-    CumulativeTimer(const CumulativeTimer& t)
```

Comment:
@afishbeck this wasn't called, and is the same as the default copy constructor, except it left logLevel uninitialized.

Reply:
looks fine